### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by laulpogan**
Addition of SECURITY.md.

Related to: [Security PR 5239](https://github.com/datahub-project/datahub/pull/5239)

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by laulpogan — COMMENTED** [review]
Previously were too many h1 headers

**Review by laulpogan — COMMENTED** [review]
cutting mailto, it caused an error

**Comment by laulpogan** [comment]
@maggiehays this is erroring because it wants to be added to the sidebar... how do I opt out of this test?

**Comment by maggiehays** [comment]
@laulpogan we have a few options -- 

* Display it in the Sidebar under the [`Join the Community` section](https://github.com/acryldata/datahub/blob/master/docs-website/sidebars.js#L461)
* Hard-code ignoring the file path in [`generateDocsDir.ts`](https://github.com/acryldata/datahub/blob/master/docs-website/generateDocsDir.ts#L109)
* Add it to `sidebar.js` & comment it out https://github.com/acryldata/datahub/blob/master/docs-website/sidebars.js#L32

If we're seeing other OSS projects surface their security policy via their hosted docs site, let's go with the first one. 

If not, I'll defer to @hsheth2 about best practices/considerations between the 2nd & 3rd options

**Comment by laulpogan** [comment]
> If we're seeing other OSS projects surface their security policy via their hosted docs site, let's go with the first one.
> 
> If not, I'll defer to @hsheth2 about best practices/considerations between the 2nd & 3rd options

It's not present in any of the major projects like grafana, mongodb, etc. I think it's a good shout to just exclude it in the dir. Will add @hsheth2 as a reviewer after I make change

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Reporting security issues
+
+If you think you have found a security vulnerability, please send a report to security@datahubproject.io. This address can be used for all of Acryl Data’s open source and commercial products (including but not limited to DataHub and Acryl Data). We can accept only vulnerability reports at this address.
+
+Please encrypt your message to us; please use our PGP key. The key fingerprint is:
```
Comment (hsheth2) [inline SECURITY.md:5]: We probably want PGP encryption to be optional and not required - looks like that's what kubernetes does

**Review by hsheth2 — COMMENTED** [review]
LGTM - minor q about pgp encryption

We probably want to merge this PR to the main datahub-project/datahub repo instead of merging here directly

File: SECURITY.md (hunk 1 above)
Comment (laulpogan) [reply SECURITY.md:5]: ah ok! I'll go ahead and make that change - I'll just open a new PR for datahub-project/datahub

**Comment by hsheth2** [comment]
Moved to upstream

**Pull request closed without merging**